static/js/components/elements/tooltip/tooltip.tsx

```diff
@@ -689,6 +748,8 @@ export const Tooltip = ({
   placement,
   followCursor = false,
   showArrow = false,
+  triggerTouchOnLongPress,
+  longPressDelay = TOOLTIP_DEFAULT_LONG_PRESS_DELAY,
```

Comment:
Out of curiosity, is there a way to better support a user's device-side accessibility settings? I'm thinking about the case where a user sets a longer-than-usual long-press duration in their phone's settings. The current approach would be locked into our set TOOLTIP_DEFAULT_LONG_PRESS_DELAY, right?

I don't think this should block this PR, just curious if you had thoughts on this.